modules/ROOT/pages/verify-nfs.adoc

```diff
@@ -0,0 +1,187 @@
+= Verify NFS Server Requirements
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Private Cloud Edition supports the Network File System (NFSv4) protocol.
+
+== Verify the Required Ports are Open
+
+The NFS server requires ports 2049 and 111 to be open.
+ 
+Use the netcat tool to verify the required ports are open. The netcat tool is a computer networking utility used to read and write to network connections using TCP or UDP. 
+
+To verify the required ports are open, execute the following commands:
```

Comment:
"that" would be helpful here as a clue to reader that the sentence structure is To verify X, not To verify the required ports (and therefore mildly confused when they hit "are open." I used to delete all "that" but here it seems helpful?